### ISSUE TITLE
Manual Event trigger payload incorrect shape

### ISSUE THREAD

**Issue by mlanders** [hasura/graphql-engine#7165]
In Hasura v2.0.1 the payload for a manual event doesn't match the [docs](https://hasura.io/docs/latest/graphql/core/event-triggers/payload.html) so not sure if expected or undocumented change.
For Manual only under data I'm getting `row` instead of `old` and `new`.

```{
  "event": {
      "session_variables": <session-variables>,
      "op": "MANUAL",
      "data": {
          "old": null,
          "new": <column-values>
      }
  },
  ```

However it's showing up like this:

```
{
    "payload": {
        "event": {
            "session_variables": null,
            "op": "MANUAL",
            "data": {
                "row": {
                    "userID": "some_value_here",
                }
            },
            "trace_context": null
        },
        ```

**Comment by aaronhayes** [comment]
Same issue with version `2.0.4`

**Comment by kolharsam** [comment]
@mlanders @aaronhayes this has been addressed in the latest release of `v2.0.7`. Thanks for reporting the issue and for your patience!
